.gitignore

```diff
@@ -78,6 +78,7 @@ src/Win32/*
 /3rdparty/glog/vsprojects/libglog_static/Release/*
 /3rdparty/rpcz/vsprojects/Intermediate/*
 /3rdparty/rpcz/vsprojects/Win32/*
+/3rdparty/protobuf/python/*
```

Comment:
It is scary to ignore all files under 3rdparty/protobuf/python, some of them are NOT auto-generated. It will be a big issue if we decide to upgrade to the next protobuf version. A better way of ignoring auto-generated protobuf files would be to list them explicitly (or to list all auto-generated folders).